src/view/UserphraseView.cpp

```diff
@@ -71,14 +79,18 @@ void UserphraseView::addPhrase(int result)
     qDebug() << "Add" << *phrase.get() << "(" << *bopomofo.get() << ")";
 
     if (dialogType_ == DIALOG_MODIFY) {
-        remove();
+        remove(QDialog::Accepted);
```

Comment:
@YWJamesLin : forward the signal instead.
Reference: https://woboq.com/blog/how-qt-signals-slots-work.html